integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java

```diff
@@ -105,6 +105,17 @@ public void testInsertWithoutTime() throws SQLException, InterruptedException {
     st1.close();
   }
 
+  @Test
+  public void testInserthaventTime() throws SQLException, InterruptedException {
```

Comment:
I can't understand the method name, is this a typo?

Reply:
I think  have not = haven’t = Without. I will change it.